src/main/java/me/figo/FigoConnection.java

```diff
@@ -302,26 +194,7 @@ public void revokeToken(String token) throws IOException, FigoException {
      * @return Auto-generated recovery password
      */
     public String addUser(String name, String email, String password, String language) throws IOException, FigoException {
-        CreateUserResponse response = this.queryApi("/auth/user", new CreateUserRequest(name, email, password, language), "POST",
-                CreateUserResponse.class);
+        CreateUserResponse response = this.queryApi("/auth/user", new CreateUserRequest(name, email, password, language), "POST", CreateUserResponse.class);
         return response.recovery_password;
     }
-    
-    /**
-     * Creates a new figo User and returns a login token
-     * @param name
-     *            First and last name
-     * @param email
-     *            Email address; It must obey the figo username and password policy
-     * @param password
-     *            New figo Account password; It must obey the figo username and password policy
-     * @param language
-     *            Two-letter code of preferred language
-     * @return TokenResponse for further API requests
-     */
-    public TokenResponse addUserAndLogin(String name, String email, String password, String language) throws IOException, FigoException {
```

Comment:
And there are customer use cases of the API where that is totally fine. If you consider this call dangerous, then please mark it, but not just remove it as others might find it useful (as in this case)